flytekit/models/literals.py

```diff
@@ -700,6 +700,40 @@ def from_flyte_idl(cls, pb2_object):
         return cls({k: Literal.from_flyte_idl(v) for k, v in pb2_object.literals.items()})
 
 
+class OutputData(_common.FlyteIdlEntity):
```

Comment:
maybe you should add a new property called 
```
@property
def literals() -> Dict[str, Literal]:
  ...
```
This is to mimic - https://github.com/flyteorg/flytekit/blob/452a2ecde0bf4f9ddd1ed25dc8b446f1ad0e3e7d/flytekit/models/literals.py#L681

Otherwise, things like this will be missed IMO 
https://github.com/flyteorg/flytekit/blob/452a2ecde0bf4f9ddd1ed25dc8b446f1ad0e3e7d/flytekit/remote/remote.py#L1880

you have handled it fine here as you convert to literalmap